.github/workflows/release-dotnet.yml

```diff
@@ -2,25 +2,62 @@ name: Release .NET SDK
 
 on:
   workflow_dispatch:
-  release:
-    types: [published]
+  # release:
+  #   types: [published]
+  workflow_run:
+    workflows: ["Release Platform Libraries"]
+    branches: [main]
+    types:
+      - completed
```

Comment:
oooh, that's cool! Can we add a manual trigger as well for parity with other actions?

Reply:
Yeah, there's `workflow_dispatch` trigger above.